lib/main.coffee

```diff
@@ -108,6 +108,7 @@ module.exports =
         parameters = @parameters.filter (item) -> item
         command = @command
         confFile = helpers.find(path.dirname(filePath), ['phpcs.xml', 'phpcs.xml.dist', 'phpcs.ruleset.xml'])
+        cwd = path.dirname(confFile) if confFile
```

Comment:
You wouldn't need to wrap the whole `exec` call in an `if`, just move the options into a variable and put the adding of that parameter in an `if`.

If the behavior is the same I guess it's not that big of a deal, other than showing it was something that was thought about instead of an oversight.